Add link to notification docs in privilege docs (#589)
neo4j/docs-cypher: modules/ROOT/pages/administration/access-control/database-administration.adoc

Use `REVOKE` if you want to remove a privilege.
====

> Common errors such as misspellings or attempting to grant privileges to roles where either does not exist will lead to notifications.

Reviewer: granting to non-existing will fail not give notifications, I guess it should say revoke?

> Common errors such as misspellings or attempting to grant privileges to roles where either does not exist will lead to notifications.

Reviewer: either what?
```
to roles where either does not exist
```

> Common errors such as misspellings or attempting to revoke a privileges that have not been granted or denied will lead to notifications.

Reviewer 2: ```suggestion
Common errors, such as misspellings or attempts to revoke privileges that have not been granted or denied, will lead to notifications.
```

> Some of these notifications may be replaced with errors in a future major version of neo4j.

Reviewer: Either we only mention the cases that'll give errors or we also mention the cases that'll stay notifications, right now it feels like we have a combo with this sentence (giving only the error cases and then saying some as if not all error cases will be throwing errors later)

Author: I think we could leave this non-specific. As long as the details are described on the notifications page.

> Some of these notifications may be replaced with errors in a future major version of neo4j.

Reviewer 2: ```suggestion
Some of these notifications may be replaced with errors in a future major version of Neo4j.
```

> See link:{neo4j-docs-base-uri}/status-codes/{page-version}/notifications/all-notifications[Status Codes -> Neo4j Status Codes v5 -> Notification codes] for details on Notifications.

Reviewer 2: ```suggestion
See link:{neo4j-docs-base-uri}/status-codes/{page-version}/notifications/all-notifications[Status Codes -> Notification codes] for details on notifications.
```
